custom_components/homewhiz/sensor.py

```diff
@@ -81,7 +76,7 @@ def native_unit_of_measurement(self):
         if "TEMP" in self.key:
             return TEMP_CELSIUS
         if "SPIN" in self.key:
-            return "rpm"
+            return "RPM"
```

Comment:
The other commented function returns `" RPM"` (with space)

Reply:
Done